docs/notes-KZG10.md

```diff
@@ -1,3 +1,74 @@
 In this module we show how and why the KZG10 
-polynomial commitment scheme has been modularised 
-for this PLONK implementation.
\ No newline at end of file
+polynomial commitment scheme has been implemented
+for this PLONK implementation.
+
+KZG10 Commitments 
+==================
+
+PLONK can be constructed with different 
+commitment schemes and does not requre solely
+homomorphic commitments. However, this library
+has only homomorphic commitment schemes as they 
+are intelligble for users and have many useful
```

Comment:
What does this part mean? I don't quite understand what you mean by "this library has only homomorphic commitment schemes as they are intelligble for users"

Reply:
Well I mean for us having an intelligible code rather than having something that is out of line with the steps of PLONK. I can change it to add a bit more clarity.

Reply:
Sorry I'm a bit slow today, so i'm still not following. What do you mean by "something that is out of line with the steps of PLONK" ?

Reply:
The steps of PLONK use the lineariser from sonic which requires homomorphism on the commitments. 
If it doesn't read well for you, then i'll change it.

Reply:
Do you mean that not using a homomorphic commitment scheme will deviate from the original proving/verification procedure shown in the PLONK paper?

And that changing to a non-homomorphic commitment scheme will make the code un-intelligible to users?

Reply:
Yes to the first question. Sort of to the second part, in that it will not be in line with what those following PLONK paper will expect.  It shouldn't matter anymore because I have refactored the expression, so if you can just look at it when I request the review again then it should be good 👍